sso-service-migration.md

```diff
@@ -0,0 +1,58 @@
+## SSO Service Name Migration
+`rh-sso-admin` automated the SSO internal changes needed to enable a new service name with BCGov IDPs.
+
+### Preparation:
+1. take a manual backup of SSO DB, in case where we need to roll back:
```

Comment:
Likely fine - but do we want to run a verification step for the backup? (Although Wade's stuff is tight!)

Reply:
ah good point! The only issue is that we are not taking down the sso route this time (will need it there for API requests) so it'll cause conflicts if we restore. So far I'm pretty confident about the backup container as we tested during upgrade. What do you think?

Reply:
Fully agree - our team has not had any issues with our backups either. I'll just note that the verification function `backup.sh -v all` spins up a local server and attempts a restore there, so it shouldn't impact the existing instance. His docs:

> Verify Options:
  ================
    The verify process performs the following basic operations:
      - Start a local database server instance.
      - Restore the selected backup locally, watching for errors.
      - Run a table query on the restored database as a simple test to ensure tables were restored
        and queries against the database succeed without error.
      - Stop the local database server instance.
      - Delete the local database and configuration.